locations/spiders/bmw_es.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+import scrapy
+import xmltodict
+
+from locations.dict_parser import DictParser
+
+
+class BeerBmwEsSpider(scrapy.Spider):
+    name = "bmw_es"
+    item_attributes = {
+        "brand": "bmw",
+        "brand_wikidata": "Q26678",
+    }
+    allowed_domains = ["bmw.es"]
+    start_urls = [
+        "https://www.bmw.es/c2b-localsearch/services/api/v4/clients/BMWSTAGE2_DLO/ES/pois?brand=BMW_BMWI_BMWM&cached=off&category=BM&country=ES&language=es&lat=0&lng=0&maxResults=700&showAll=true&unit=km"
+    ]
+
+    def parse(self, response):
+        data_dict = xmltodict.parse(response.text)
+        json_data = json.loads(json.dumps(data_dict))
```

Comment:
There's no need to pass the data through json.loads/json.dumps.

```suggestion
        json_data = xmltodict.parse(response.text)
```